### PR TITLE
IHapticDevice interface added

### PR DESCRIPTION
**Pull request by aerydna**
imported the IHapticDevice interface from https://github.com/robotology/haptic-devices.

### CODE REVIEW

File: src/libYARP_dev/include/yarp/dev/IHapticDevice.h [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+ */
+
+#ifndef __HAPTICDEVICE_INTERFACE__
+#define __HAPTICDEVICE_INTERFACE__
+
+#include <yarp/sig/Vector.h>
+#include <yarp/sig/Matrix.h>
+
+namespace hapticdevice {
+    /**
```
Comment (barbalberto) [inline src/libYARP_dev/include/yarp/dev/IHapticDevice.h]: Why namespace hapticdevice?

Comment (pattacini) [inline src/libYARP_dev/include/yarp/dev/IHapticDevice.h]: @barbalberto it was already fixed via 03d1208020c151456f871c1872dee3be3979ccbb.

File: src/libYARP_dev/include/yarp/dev/IHapticDevice.h [hunk 2]
```diff
@@ -0,0 +1,116 @@
+// -*- mode:C++; tab-width:4; c-basic-offset:4; indent-tabs-mode:nil -*-
+
+/*
+ * Copyright (C) 2015 iCub Facility - Istituto Italiano di Tecnologia
+ * Author: Ugo Pattacini
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later.
+ *
+ */
+
+#ifndef __HAPTICDEVICE_INTERFACE__
```
Comment (francesco-romano) [inline src/libYARP_dev/include/yarp/dev/IHapticDevice.h]: header guards should follow the YARP convention

Comment (francesco-romano) [inline src/libYARP_dev/include/yarp/dev/IHapticDevice.h]: @pattacini already fixed it... Thanks @pattacini

**Pull request merged**